surveys/typeform.py

```diff
@@ -133,3 +141,5 @@ def sync_learner_responses():
             for attr, value in data.items():
                 setattr(survey_response, attr, value)
             survey_response.save()
+
+        update_course_community_feedback(study_group)
```

Comment:
Same. Actually, please merge the tasks to look like:

```python
@shared_task
def sync_surveys():
    new_facilitator_responses = sync_facilitator_responses()
    new_learner_responses = sync_learner_responses()
    affected_courses = set([r.study_group.course for r in new_facilitator_responses + new_learner_responses])
    for courses in affected_courses:
        course.calculate_ratings()
        course.calculate_tagdorsements()
```
You'll just need to return the new responses from `sync_facilitator_responses` and `sync_learner_responses`